src/App.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-import './App.css';
 import SigninForm from './components/signinForm';
 import SignupForm from './components/signupForm';
+import { BrowserRouter as Router,Switch, Route } from 'react-router-dom';
 
 function App() {
   return (
-    <div>
-    <SigninForm/>
-  
-    </div>
+    <Router>
+      <Switch>
+        <Route exact path="/"
+        component={SignupForm}/>
+        <Route path="/signinForm"
```

Comment:
It's better to use a verb for the route path like `/signin` and not `signinForm`

Reply:
Okay sir